panther_analysis_tool/main.py

```diff
@@ -16,6 +16,7 @@
 You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
+#  pylint: disable=too-many-lines
```

Comment:
I'm realizing that the PR I merged yesterday might cause some "fun" merge conflicts here. 😕